bindings/wasm/src/account/types/cek_algorithm.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use identity::account_storage::CekAlgorithm;
+use serde::Deserialize;
+use serde::Serialize;
+use wasm_bindgen::prelude::*;
+
+use crate::account::types::WasmAgreementInfo;
+use crate::error::Result;
+use crate::error::WasmResult;
+
+/// Supported CEK algorithms
+#[derive(Clone, Serialize, Deserialize)]
+#[wasm_bindgen(js_name = CekAlgorithm, inspectable)]
+pub struct WasmCekAlgorithm(CekAlgorithm);
+
+#[wasm_bindgen(js_class = CekAlgorithm)]
+impl WasmCekAlgorithm {
+  /// ECDH-ES will be used as the content encryption key.
```

Comment:
I would suggest my suggestion for the documentation for `CekAlgorithm::EcdhEs` here.